README.md

```diff
@@ -95,10 +95,9 @@ Ape's modular plugin system allows users to have an interoperable experience wit
 
 - Learn more about **developing** your own plugins from this [developing user guide](https://docs.apeworx.io/ape/stable/userguides/developing_plugins.html).
 
-```{note}
-If a plugin does not originate from the [ApeWorX GitHub Organization](https://github.com/ApeWorX?q=ape&type=all), you will get a warning about installing 3rd-party plugins.
-Install 3rd party plugins at your own risk.
-```
+> \[!NOTE\]
```

Comment:
Oh nice! Is this specific to Github markdown, or does it also work with sphinx-markdown?

Reply:
This one is specific for Github markdown. I don't think it works the same with sphinx-markdown.

Reply:
Remember that this is the quickstart guide for our docs as well

Reply:
![image](https://github.com/user-attachments/assets/ec87268b-79cd-4328-9b1f-857d841f9e7c)

This will render in the docs like this which I think is still a good way of doing this since it makes it more clear on github about the note and is still clear enough when rendered with sphinx.

Reply:
Personally I love the note highlighting we have in the docs now, I would love to have it render in github better but I think the docs rendering takes precedence over github